### PR TITLE
*: add some faq (#523)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #523 to release-1.0

---

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by CharLotteiu** [comment]
Conflicts resovled. @lance6716 @yikeke PTAL.

**Comment by lance6716** [comment]
LGTM

File: zh/faq.md [hunk 1]
```diff
@@ -141,11 +148,29 @@ DM 在最后 `rename ghost_table to origin table` 的步骤会把内存的 DDL 
...
+
+```
+if the DDL is not needed, you can use a filter rule with \"*\" schema-pattern to ignore it.\n\t : parse statement: line 1 column 11 near \"EVENT `event_del_big_table` \r\nDISABLE\" %!!(MISSING)(EXTRA string=ALTER EVENT `event_del_big_table` \r\nDISABLE
+```
+
+出现报错的原因是 TiDB parser 无法解析上游的 DDL，例如 `ALTER EVENT`，所以 `sql-skip` 不会按预期生效。可以在任务配置文件中添加 [Binlog 过滤规则](feature-overview.md#binlog-event-filter)进行过滤，并设置 `schema-pattern: "*"`。
+
+## DM 同步时下游长时间出现 REPLACE 语句
+
+请检查是否符合 [safe mode 触发条件](glossary.md#safe-mode)。如果任务发生错误并自动恢复，或者发生高可用调度，会满足“启动或恢复任务的前 5 分钟“这一条件，因此启用 safe mode。
```
Comment (lance6716) [inline zh/faq.md:174]:
```suggestion
请检查是否符合 [safe mode 触发条件](glossary.md#safe-mode)。如果任务发生错误并自动恢复，会满足“启动或恢复任务的前 5 分钟“这一条件，因此启用 safe mode。
```

v1.0没有这个高可用特性

**Comment by lance6716** [comment]
LGTM

**Review by leoppro — APPROVED** [review]

**Review by yikeke — APPROVED** [review]

**Pull request merged**